api/v1alpha3/awscluster_webhook_test.go

```diff
@@ -389,8 +431,14 @@ func TestAWSCluster_DefaultAllowedCIDRBlocks(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			tt.beforeCluster.Default()
-			g.Expect(tt.beforeCluster.Spec.Bastion).To(Equal(tt.afterCluster.Spec.Bastion))
+			ctx := context.TODO()
+			cluster := tt.beforeCluster.DeepCopy()
+			cluster.ObjectMeta = metav1.ObjectMeta{
+				GenerateName: "cluster-",
+				Namespace:    "default",
+			}
+			g.Expect(testEnv.Create(ctx, cluster)).To(Succeed())
```

Comment:
Does this call Default()?

Reply:
envtest's API Server is calling the running webhook

Reply:
right, and Defaulting is called during Create right? Forgot if it is only being called during get.

Reply:
It's called on Create or Update depending on the kubebuilder verbs. Only conversion webhooks are called on get.